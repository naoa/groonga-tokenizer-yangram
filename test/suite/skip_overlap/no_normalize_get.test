register tokenizers/yangram

yangram_register 2 --skip_overlap 1
tokenize TokenYaBigramSkipOverlap "今日は、雨だな。" --mode GET
