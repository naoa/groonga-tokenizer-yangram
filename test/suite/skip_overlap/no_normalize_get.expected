register tokenizers/yangram
[[0,0.0,0.0],true]
tokenize TokenYaBigram "今日は、雨だな。" --mode GET
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "今日",
      "position": 0,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "は、",
      "position": 2,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "雨だ",
      "position": 4,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "な。",
      "position": 6,
      "force_prefix": false,
      "force_prefix_search": false
    }
  ]
]
